core/components/molecules/chipInput/ChipInput.tsx

```diff
@@ -225,6 +225,7 @@ export const ChipInput = (props: ChipInputProps) => {
         className="my-3 mx-2"
         onClick={() => onClick && onClick(chip, index)}
         onClose={() => onChipDeleteHandler(index)}
+        maxWidth="var(--spacing-8)"
```

Comment:
There is no need to pass `maxWidth` prop to Chip as it is already accommodated in the `{...rest}` through `chipOptions` prop